### ISSUE TITLE
In example app HW Acceleration cuda doesn't work

### ISSUE THREAD

**Issue by Symbai** [Ruslan-B/FFmpeg.AutoGen#176]
* **I'm submitting a ... **
  - [X] bug report
  - [ ] feature request
  - [ ] support request or question => Please do not submit support request or questions here, see note at the top of this template.

* **What is the current behavior?**
I have not modified any code. Selecting CUDA (I have a nvidia geforce 2080TI) in example application throws with `Could not initialize the conversion context`
https://github.com/Ruslan-B/FFmpeg.AutoGen/blob/40121f748d66fdde604ea53da05c7f883953c851/FFmpeg.AutoGen.Example/VideoFrameConverter.cs#L31

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem*:
1. Download master branch
2. Run example application
3. Select CUDA
4. => Exception

* **What is the expected behavior?**
I expect the example application to work with cuda hardware acceleration, especially because it asks me if I want to use it.

* **What is the motivation / use case for changing the behavior?**
The example application should work out of the box

* **Please tell us about your environment:**

- version: master branch as of now

I can use cuda without a problem on ffmpeg binaries from https://github.com/BtbN/FFmpeg-Builds/releases/download/autobuild-2021-10-03-12-23/ffmpeg-n4.4-154-g79c114e1b2-win64-gpl-shared-4.4.zip I've also replaced the dll files from this archive (because its built with `--enable-cuda-llvm`) and run example app again but same error.

**Comment by Ruslan-B** [comment]
I guess you have to compile FFmpeg binaries yourself - this repo binaries are used for API generation only. Besides, binaries in repo  are GPL - I do not compile and ship them. The source is https://www.gyan.dev/ffmpeg/builds/ so you can check if they were compiled with or without cuda support.

**Comment by Ruslan-B** [comment]
Besides, your pc supposed to have cuda libraries installed -as they are nor not part of standard drivers - to many factors, hope it helps

**Comment by Symbai** [comment]
> The source is https://www.gyan.dev/ffmpeg/builds/ so you can check if they were compiled with or without cuda support.

Please read my post again, especially the last part.

**Comment by Ruslan-B** [comment]
Have you read my comment?

**Comment by Symbai** [comment]
I wrote that ffmpeg was built with cuda enabled. I wrote that ffmpeg (the executable) works with cuda. I wrote that my support supports cuda. I wrote that YOUR example doesn't work with cuda.

While I was trying to provide as much as possible all you did was telling me that I need special ffmpeg build and cuda driver. You have proven that you didn't read fully read my post where I said I have both. This is really unfriendly and that you're closing this is unfriendly as well.

Have you actually tested your example on a system with cuda? //edit Seeing someone else linked to this issue means I'm not alone.
